web/pages/group/[...slugs]/index.tsx

```diff
@@ -64,14 +64,6 @@ export async function getStaticPropz(props: { params: { slugs: string[] } }) {
   const memberIds = group && (await listMemberIds(group))
   const creatorPromise = group ? getUser(group.creatorId) : null
 
-  const contracts =
-    (group && (await listContractsByGroupSlug(group.slug))) ?? []
-  const now = Date.now()
-  const suggestedFilter =
-    contracts.filter((c) => (c.closeTime ?? 0) > now).length < 5
-      ? 'all'
-      : 'open'
-
```

Comment:
This logic was kind of good for groups. Means most long running groups default to open filter.

Also, we wanted the default sort to be trending.

It's worth keeping these IMO.

Reply:
Appears that Relevance is equivalent to Trending with no query, so that's reasonable then.

I guess the open filter is mostly unnecessary when sorting by Trending as well, so I retract my comment.